package.json

```diff
@@ -8,16 +8,20 @@
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
     "babel-preset-env": "^1.6.0",
     "core-js": "^2.5.1",
+    "coveralls": "^3.0.0",
     "escodegen": "^1.9.0",
     "eslint": "^4.10.0",
     "eslint-plugin-mozilla": "^0.4.9",
     "eslint-plugin-no-unsanitized": "^2.0.1",
     "fancy-log": "^1.3.2",
     "gulp": "^3.9.1",
+    "gulp-inject": "^4.3.0",
```

Comment:
The `gulp-inject` dependency is no longer used.

Reply:
removed